### PR TITLE
Add new tool PSMValidate

### PR DESCRIPTION
**Pull request by tmcgowan**
Validate identified PSMs by comparing to peptide ion fragmentation

### CODE REVIEW

File: tools/psm_validation/psm_validation.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        <param name="peptide_sequences" type="data" format="tabular" label="Peptide Sequence List" />
+        <param name="b_ion_run" type="integer" value="2" label="Consecutive B-Ions" />
+        <param name="y_ion_run" type="integer" value="2" label="Consecutive Y-Ions" />
+        <param name="neutral_loss_ions" type="boolean" checked="True" label="Use Neutral Loss" />
+        <param name="internal_ions" type="boolean" checked="True" label="Use Internals" />
+        <param name="epsilon" type="float" label="Epsilon">0.5</param>
+    </inputs>
+    <outputs>
+        <data name="html_output" format="html" from_work_dir="index.html" label="PSM Ion Fragementation Report" />
+        <data name="text_output" format="csv" from_work_dir="results.csv" label="CSV Results" />
```
Comment (bgruening) [inline tools/psm_validation/psm_validation.xml:37]: @tmcgowan can we make sure this is a tabular formatted file? `tabular`?

Comment (tmcgowan) [reply tools/psm_validation/psm_validation.xml:37]: The outputs are HTML and CSV. Neither is tabular. I am sorry, but I do not understand your request.

Comment (bgruening) [reply tools/psm_validation/psm_validation.xml:37]: something like that `sed -e 's/,/\t/g' /tmp/file.csv > /tmp/file.tabbed` after the tools has run ... Galaxy has 100 of tools capable of handling tabular files - for csv we need to convert them in a separate step to tabular to make use of those tools. This is the reason I suggerst to let your tool spit out a tabular file natively. does that make sense?

Comment (tmcgowan) [reply tools/psm_validation/psm_validation.xml:37]: OK, now I understand.

**Review by bgruening — APPROVED** [review]
Cool, thanks a lot!

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**